gobblin-data-management/src/main/java/gobblin/data/management/copy/CopyConfiguration.java

```diff
@@ -12,29 +12,44 @@
 
 package gobblin.data.management.copy;
 
+import lombok.AllArgsConstructor;
 import lombok.Data;
 
 import org.apache.hadoop.fs.Path;
 
+import com.google.common.base.Optional;
+
 
 /**
  * Configuration for Gobblin distcp jobs.
  */
 @Data
+@AllArgsConstructor
 public class CopyConfiguration {
```

Comment:
Can we refactor this to use a builder, and be able to build from options? `DESTINATION_GROUP_KEY` and `PRESERVE_ATTRIBUTES_KEY` should be members of this class instead of `CopySource`.